Documentation/gettingstarted.rst

```diff
@@ -1013,33 +1016,43 @@ The output should simply return:
     OK
 
 
-Run a script to create the client requests to the newly created web-server. The script should generate a ``client.json`` for the client task:
+Run a script to create "good client" and "bad client" requests to the newly created web-server. The script should generate ``goodclient.json`` and ``badclient.json`` files for the client tasks, respectively:
```

Comment:
Unfortunately, the scripts are in the archive so it is not easy to comment on them :(

For generate_client_file.sh I'd probably add a check to verify that is either goodclient or badclient only. Otherwise, a simple typo might mess up the demo.